vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+olccreated_flag: "{{ flags_dir }}/.olccreated.flag"
+olc_tmp: /tmp/olc.conf
```

Comment:
Doch nicht: ` In case you need to make them world-accessible you need to use file module.`